specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/ConversationalAnalysis-authoring.json

```
@@ -2441,17 +2663,16 @@
         "createdDateTime",
         "lastUpdatedDateTime",
         "expirationDateTime",
-        "status",
-        "errors"
+        "status"
       ],
       "type": "object",
       "properties": {
         "result": {
-          "description": "Represents Training Tasks Detailed Result.",
+          "description": "Represents training tasks detailed result.",
           "$ref": "#/definitions/TrainingJobStatusResult"
         },
         "jobId": {
-          "description": "Gets or sets the jobId.",
+          "description": "Gets or sets the job id.",
```

```suggestion
          "description": "Gets or sets the job ID.",
```

Check elsewhere as well, in case I missed one. ID is a pseudo-acronym and should be capitalized.